internal/pkg/config/v1/local_ports.go

```diff
@@ -1,8 +1,8 @@
 package v1
 
 type LocalPorts struct {
```

Comment:
Do we want to call this struct `out`, and move it to where it's used (command_kafka_start.go) to match the style of our other commands? We may want to add additional fields to this table in the future. As far as I can tell, this has nothing to do with the configuration file, so it shouldn't belong in internal/pkg/config.

Reply:
this is being saved into config.json, and other commands get it from config as well

Reply:
> this is being saved into config.json

Why does it need be stored if we have flags? When does it get stored in the first place?